src/views/pages/_package.html

```diff
@@ -240,5 +240,12 @@
 				}
 			}
 		},
+		onrender () {
+			if (!Ractive.isServer) {
```

Comment:
This seems wrong. Why do we need it? If the browser remembers a scroll position after e.g. page refresh, it shouldn't jump back to the top.

Reply:
You're right, this was supposed to prevent the page from incorrect position after navigating from the search results, but I missed that it would work every time, even when the page reloaded. Fixed.